flocker/common/functional/test_era.py

```diff
@@ -0,0 +1,31 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Functional tests for ``flocker-node-era`` command.
+"""
+
+from unittest import skipUnless
+from subprocess import check_output
+
+from twisted.python.procutils import which
+
+from .._era import get_era
+from ...testtools import make_script_tests
+
+EXECUTABLE = b"flocker-node-era"
+
+
+class FlockerNodeEraTests(make_script_tests(EXECUTABLE)):
+    """
+    Tests for ``flocker-node-era``.
+    """
+    @skipUnless(which(EXECUTABLE), EXECUTABLE + " not installed")
```

Comment:
Flocker doesn't work on non-Linux. The fact you can install it is merely a convenience for developers. This seems like a waste of time to me.